x-pack/plugins/lens/public/indexpattern_datasource/dimension_panel/time_scaling.tsx

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiLink,
+  EuiFormRow,
+  EuiSelect,
+  EuiFlexItem,
+  EuiFlexGroup,
+  EuiButtonIcon,
+  EuiText,
+  EuiPopover,
+  EuiButtonEmpty,
+  EuiSpacer,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React, { useState } from 'react';
+import { StateSetter } from '../../types';
+import { IndexPatternColumn, operationDefinitionMap } from '../operations';
+import { mergeLayer } from '../state_helpers';
+import { unitSuffixesLong } from '../suffix_formatter';
+import { TimeScaleUnit } from '../time_scale';
+import { IndexPatternPrivateState } from '../types';
+
+const DEFAULT_TIME_SCALE = 'm' as const;
```

Comment:
Why is this the default, and not per-second? I ask because we can pretty much guarantee that per-second is smaller than the interval they have, so it's unlikely to scale up. I think that's a benefit.

Reply:
Took that from the mocks as well, but I see your point and I think per second is a better default